### ISSUE TITLE
Orange led does not blink when charging

### ISSUE THREAD

**Issue by Massimaux** [ComThings/PandwaRF#101]
Hi,

I have used the pandwaRF for year and a half and it has worked very well up till today. The battery was low and I started charging the dongle. But, there was no orange led blinking. Also, the pandwaRF app cannot see it when scanning.

It has been charging for more than an hour but the only thing happening is short simultaneous orange and green blinks about every minute.

What is going on? How can I check if the dongle is still functional?

Thanks

**Comment by trollwookiee** [comment]
Hi, 
few questions:

- is the USB power plugged into PandwaRF while scanning with the app ?
- when you connect the PandwaRF to your smartphone using USB cable, is it detected by the app ?
- a possibility is that the battery is dead

**Comment by Massimaux** [comment]
> * is the USB power plugged into PandwaRF while scanning with the app ?

Yes. If the battery is dead and if PandwaRF is functional, is this case possible?

> * when you connect the PandwaRF to your smartphone using USB cable, is it detected by the app?

No.

**Comment by Massimaux** [comment]
@trollwookiee 

If the battery is removed and the USB power is plugged in, can the PandwaRF work?

**Comment by trollwookiee** [comment]
Yes, PandwaRF can work without battery

**Comment by Massimaux** [comment]
> Yes, PandwaRF can work without battery

Thanks. Since the PandwaRF does not work when plugged with USB power, can we conclude that the dongle has failed?

**Comment by trollwookiee** [comment]
If all LEDs are OFF when plugged in USB, then the board is probably dead
